app/views/submissions/show.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="show_and_edit">
+  <% if post.text.blank? %>
+    <div class="tabbed">
+      <div class="label">title:</div>
+      <div><%= post.title %></div>
+    </div>
+    <div class="tabbed">
+      <div class="label">link:</div>
+      <div><%= link_to post.url %></div>
+    </div>
+  <% else %>
+    <div class="tabbed">
+      <div class="label">text:</div>
+      <div><%= post.text %></div>
+    </div>
+  <% end %>
+  <div class="tabbed">
+    <div class="label">contributer:</div>
+    <div><%= post.user.name %></div>
+  </div>
+  <div class="tabbed">
+    <div class="label">submitted at:</div>
+    <div><%= time_ago_in_words(post.created_at) %> ago </div>
+  </div>
+</div>
+<div class="comments-posted">
+  <% if post.comments.exists? %>
+    <% post.comments.each do |c| %>
+      <%= "comment by #{c.user.name}: #{c.text}" %>
+      <br />
+    <% end %>
+  <% end %>
+</div>
+<%= form_for(new_comment, url: submission_create_comment_path(post.id), method: :post) do |f| %>
```

Comment:
I'm tempted to suggest removing the `new_comment` method in favor of inlining with `current_user.submissions.build`.

Also, should just be able to pass `post` into `submission_create_comment_path`.